index.html

```diff
@@ -2794,6 +2794,13 @@ <h3 id="lifetime-AudioNode" class="informative">
             <em>playing</em> reference to themselves while they are currently
             playing.
             </li>
+            <li>An <dfn>active reference</dfn> for
+            <a><code>AudioWorkletNode</code></a>s. This node maintains an <em>
+              active</em> reference to itself upon creation. This reference is
+              subsequently maintained or removed based on the return values
+              from invocation of its associated <a>AudioWorkletProcessor</a>'s
+              <code>process()</code> method.
```

Comment:
I think this is correct, because this is just one kind of reference and there are several. The text prior to this list of reference kinds says, _An AudioNode will live as long as there are any references to it. There are several types of references:_

The later definition of `process()` explicitly links to the paragraph on this _active reference_ and says nothing about JS references (which must be based on application code and can't be controlled by `process()`). So I agree with you about the actual behavior, I just don't see that the documentation says anything wrong here.

Would a different name for this type of reference (not "active") make this clearer?